# ID of solution (as published in Task msg)
uint32 solution_id

---

Solution solution
